src/ui/src/builder/useComponentDescription.ts

```diff
@@ -12,13 +27,17 @@ export function useComponentDescription(
 	);
 
 	const name = computed(() => {
-		const { type, content } = unref(component);
+		const c = unref(component);
+		const { type, content } = c;
 		if (type == "html" && content?.["element"]) {
 			return content?.["element"];
 		}
 		if (type == "blueprints_blueprint") {
 			return content?.["key"] || "Blueprint";
 		}
+		if (type == "shared_blueprint") {
+			return getSourceBlueprintName(wf, c) || "Shared Blueprint";
+		}
```

Comment:
From my previous comment, that becomes not necessary.
```suggestion
		if (type == "shared_blueprint") {
			return getSourceBlueprintName(wf, c);
		}
```